scripts/processing/08-make_cov.py

```diff
@@ -30,7 +31,8 @@ def run_covariance(subject_id, tsss=False):
                             % (subject, l_freq))
     print('  Computing regularized covariance')
     epochs = mne.read_epochs(fname_epo, preload=True)
-    cov = mne.compute_covariance(epochs, tmax=0, method='shrunk')
+    cv = ShuffleSplit(3, random_state=random_state)
```

Comment:
@dengemann I think we should use k-fold as there is a lot of autocorrelation in data.